src/program/snabbvmx/doc/README.troubleshooting.md

```diff
@@ -507,34 +549,37 @@ return {
     next_hop_mac = "90:e2:ba:94:2a:bc",  <<< the next-hop mac to use for outgoing packets
   },
 }
-```  
+```
 
 **The input pcaps**
-The check app requires one or two input pcaps.
+
+The snabb "check" app requires one or two input pcaps.
 It is ok to:
+
 - only feed V4-IN.PCAP
 - only feed the V6-IN.PCAP
 - or feed both V4-IN.PCAP and V6-IN.PCAP
 
-Note for interactive tests: 
-When only feeding one pcap, then the other empty pcap must be the empty.pcap in 
-src/program/snabbvmx/tests/end-to-end/data/empty.pcap
+Note for interactive tests:<br>
```

Comment:
This tag (<br>) is not rendered as a break-line when converting the markdown file to PDF:

```
program/snabbvmx $ make snabbvmx-doc
```

A simply text break-line would work.